srcpkgs/black/update

```diff
@@ -0,0 +1 @@
+ignore="*a*"
```

Comment:
this will ignore `1.33.7-all-bugs-squashed`

Reply:
Good point, I'll push a fix for it.

Reply:
Actually, according to the manual, `ignore` is matched against version numbers, so I think this shouldn't be an issue? Many other packages also do something similar, so if it is an issue, those packages have to be updated too.

Reply:
it won't actually ignore those, as it only captures the version part of the regex it uses

Reply:
@classabbyamp see my [comment above](https://github.com/void-linux/void-packages/pull/41481#issuecomment-1373753132).

Reply:
it's pretty clear what beta would mean in the future, if used again.

Reply:
Okay then.